examples/rstudio/test_rstudio.py

```diff
@@ -0,0 +1,33 @@
+import getpass
+import time
+import selenium
+from selenium import webdriver
+from selenium.webdriver import FirefoxOptions
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+opts = FirefoxOptions()
+opts.add_argument("--headless")
+driver = webdriver.Firefox(log_path='/tmp/geckodriver.log',firefox_options=opts)
```

Comment:
Put the log in `$BATS_TMPDIR` so the filesystems stays tidy and different test runs don't clash.